magento/devdocs
# Add content to ObjectManager topic

## Purpose of this pull request

This pull request adds content to the ObjectManager topic. The content is for an important step developers must follow when programmatically updating products using core classes.

## Related issues

* MAGEDOC-3850
* KB-96

## Affected DevDocs pages

* https://devdocs.magento.com/guides/v2.3/extension-dev-guide/object-manager.html
* https://devdocs.magento.com/guides/v2.2/extension-dev-guide/object-manager.html
* https://devdocs.magento.com/guides/v2.1/extension-dev-guide/object-manager.html

## Validation Steps

1. Open the `guides/v2.1/extension-dev-guide/object-manager.md` file.
2. Verify that the raw markdown content is easier to read than the mainline version
3. Build and serve the preview server using `rake`
4. Verify no build failures linked to this topic
5. Navigate to `guides/v2.1/extension-dev-guide/object-manager.html` or the 2.2 or 2.3 version of the page.
6. Verify the new `Programmatic product updates` section is present.
